Source/ProjectFirma.Web/Views/County/Detail.cshtml

```diff
@@ -0,0 +1,150 @@
+        jQuery("#resultsTab").on("shown.bs.tab", function() {
+            _.forOwn(window.GoogleCharts.chartWrappers,
+                function (chartWrapper) {
+                    if (jQuery('#' + chartWrapper.getContainerId()).is(':visible')) {
+                        chartWrapper.getChart().clearChart();
+                        chartWrapper.draw();
+                    }
+                });
+        });
+        jQuery("#overviewTab").on("shown.bs.tab", function () {
+            var gridName = @(ViewDataTyped.BasicProjectInfoGridName.ToJS().ToHTMLFormattedString());
+            Sitka[gridName].resizeGridWidths();
+            _.forOwn(window.GoogleCharts.chartWrappers,
+                function (chartWrapper) {
+                    if (jQuery('#' + chartWrapper.getContainerId()).is(':visible')) {
+                        chartWrapper.getChart().clearChart();
+                        chartWrapper.draw();
+                    }
+                });
+        });
+    });
+</script>
+
+<div>
+    <a class="btn backButton" href="@ViewDataTyped.IndexUrl">@BootstrapHtmlHelpers.MakeGlyphIcon("glyphicon-chevron-left") Back to all @FieldDefinition.County.GetFieldDefinitionLabelPluralized() </a>
+</div>
+
+<div class="row">
+    <div class="col-lg-5">
+        <div class="panel panelFirma">
+            <div class="panel-heading panelTitle">
+                <h2>Map</h2>
+            </div>
+            <div class="panel-body">
+                <p>
+                    This map displays the location of any @FieldDefinition.Project.GetFieldDefinitionLabelPluralized() associated with this @FieldDefinition.County.GetFieldDefinitionLabel()
+                </p>
+                <div id="@ViewDataTyped.MapInitJson.MapDivID" style="height: 320px; margin: 5px 0 5px 0">
+                </div>
+                <p style="font-size: 10px">To zoom, hold down Shift and drag a rectangle.</p>
+
+                <p>@FieldDefinition.Project.GetFieldDefinitionLabelPluralized() can be associated with multiple @FieldDefinition.County.GetFieldDefinitionLabelPluralized(), but are mapped in a single location. Therefore some @FieldDefinition.Project.GetFieldDefinitionLabelPluralized() may appear outside of the @FieldDefinition.County.GetFieldDefinitionLabel() boundary.</p>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-12">
+        <div class="panel panelFirma">
+            <div class="panel-heading panelTitle">
+                <h2>@FieldDefinition.Project.GetFieldDefinitionLabelPluralized()</h2>
+            </div>
+            <div class="panel-body">
+                @Html.DhtmlxGrid(ViewDataTyped.BasicProjectInfoGridSpec, ViewDataTyped.BasicProjectInfoGridName, ViewDataTyped.BasicProjectInfoGridDataUrl, "height:300px", DhtmlxGridResizeType.VerticalResizableHorizontalAutoFit)
+            </div>
+        </div>
+    </div>
+</div>
+@*TODO Neutered Per WA DNR #1446. May decide to bring it back later
```

Comment:
we might be able to rip this comment out and the JS dealing with switching tabs.

Reply:
I took out all the tab stuff and the comment